mime-ballerina/Package.md

```diff
@@ -1,12 +1,22 @@
 ## Package Overview
 
-This package provides functions to encapsulate multiple body parts such as attachments into a single message. The
- communication of such messages follow the MIME (Multipurpose Internet Mail Extensions) specification as specified in
-  the [RFC 2045 standard](https://www.ietf.org/rfc/rfc2045.txt).
+The MIME library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
 
-> Entity refers to the header fields and the content of a message or a part of the body in a multipart entity. 
+It encapsulates multiple body parts such as attachments into a single message. The communication of such messages 
```

Comment:
Shall we add a bit more context before this. "It encapsulates multiple body parts" looks like the middle of a description.